packages/expo-updates/android/src/main/java/expo/modules/updates/db/BuildData.kt

```diff
@@ -93,9 +83,17 @@ object BuildData {
       for ((key, value) in updatesConfiguration.requestHeaders) put(key, value)
     }
     val buildData = JSONObject().apply {
-      put(UpdatesConfiguration.UPDATES_CONFIGURATION_UPDATE_URL_KEY, updatesConfiguration.updateUrl)
+      put(UpdatesConfiguration.UPDATES_CONFIGURATION_UPDATE_URL_KEY, updatesConfiguration.updateUrl.toString())
```

Comment:
we changed the way to compare build data from two map. we compared each json key/value one by one previously and now using kotlin map comparison.

here we have to convert updateUrl to string. that makes sure the arguments passing in isBuildDataConsistent to have exactly same data type. originally it's `updatesConfiguration.updateUrl: Uri` and `databaseBuildData["updateUrl"]: String`
```kotlin
  fun isBuildDataConsistent(
    updatesConfiguration: UpdatesConfiguration,
    databaseBuildData: JSONObject
  ): Boolean
```